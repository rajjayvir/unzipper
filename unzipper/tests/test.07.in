0x0d
0x6b 0x10 0x00
0x6a 0x28 0x00
0x69 0x3c 0x00
0x68 0x4e 0x00
0x67 0x5f 0x00
0x0a 0x6f 0x80
0x78 0xcf 0xc0
0x61 0xcf 0xc1
0x62 0xbf 0xc2
0x63 0xaf 0xc4
0x64 0x9f 0xc8
0x65 0x8f 0xd0
0x66 0x7f 0xe0
0x00 0x00 0x00 0x00
