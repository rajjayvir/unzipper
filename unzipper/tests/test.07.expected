0x6b : 1 0
0x6a : 2 10
0x69 : 3 110
0x68 : 4 1110
0x67 : 5 11110
0x0a : 6 111110
0x78 : 12 111111000000
0x61 : 12 111111000001
0x62 : 11 11111100001
0x63 : 10 1111110001
0x64 : 9 111111001
0x65 : 8 11111101
0x66 : 7 1111111
